internal/cmd/examplegen/main.go

```diff
@@ -9,14 +9,14 @@ import (
 
 	"github.com/goccy/go-yaml"
 
-	v1alphaExamples "github.com/nobl9/nobl9-go/internal/manifest/v1alpha/examples"
 	"github.com/nobl9/nobl9-go/internal/pathutils"
 	"github.com/nobl9/nobl9-go/manifest"
+	v1alphaExamples2 "github.com/nobl9/nobl9-go/manifest/v1alpha/examples"
```

Comment:
Is `..2` intended to be here? It looks like it might be a placeholder or a leftover.